src/services/ApiService.ts

```diff
@@ -71,12 +78,31 @@ class ApiService {
 			})
 	}
 
+	/**
+	 * Use the fetch API over HTTP to hit the check token endpoint on the server
+	 *
+	 * @returns Promise that resolves user object if successful; otherwise, Promise that resolves to empty object
+	 */
+	pingGetUsersEndpoint(): Promise<GetUsersEndpointResult[]> {
+		return fetch(`${this._apiUrl}/user`, {
+			cache: NO_CACHE,
+			method: HTTP_GET,
+		})
+			.then((resp) => resp.json())
+			.then((obj) => obj)
+			.catch((err) => {
+				this.handleError(err)
+
+				return Promise.resolve({})
```

Comment:
To match the type expected by this method signature, this should be an empty array when an error occurs

```suggestion
				return Promise.resolve([])
```